include/torch-mlir/Dialect/Torch/Utils/Utils.h

```diff
@@ -158,6 +158,10 @@ LogicalResult getPermutedType(BaseTensorType inType,
                               SmallVector<int64_t> permuteDims,
                               Type &permutedType);
 
+// Check whether the given shapes of 2 tensors are broadcastable or not.
+LogicalResult areStaticallyBroadcastCompatible(ArrayRef<int64_t> shapeA,
```

Comment:
I am actually surprised that there are no functions already available in Torch MLIR to get the shape after broadcasting or to check if broadcastable.